packages/@dcl/playground-assets/src/index.ts

```diff
@@ -1,6 +1,9 @@
 export * from '@dcl/sdk/math'
 export * from '@dcl/sdk/ecs'
 export * from '@dcl/sdk/react-ecs'
+
 import ReactEcs from '@dcl/sdk/react-ecs'
 export default ReactEcs
```

Comment:
To enable ReactEcs global for UI
```suggestion
export { ReactEcs }
```